superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -488,45 +499,89 @@ export default class FilterableTable extends PureComponent<
     // fix height of filterable table
     return (
       <StyledFilterableTable>
-        <ScrollSync>
-          {({ onScroll, scrollLeft }) => (
-            <>
-              <AutoSizer disableHeight>
-                {({ width }) => (
-                  <div>
-                    <Grid
-                      cellRenderer={this.renderGridCellHeader}
-                      columnCount={orderedColumnKeys.length}
-                      columnWidth={getColumnWidth}
-                      height={rowHeight}
-                      rowCount={1}
-                      rowHeight={rowHeight}
-                      scrollLeft={scrollLeft}
-                      width={width}
-                      style={{ overflow: 'hidden' }}
-                    />
-                    <Grid
-                      cellRenderer={this.renderGridCell}
-                      columnCount={orderedColumnKeys.length}
-                      columnWidth={getColumnWidth}
-                      height={totalTableHeight - rowHeight}
-                      onScroll={onScroll}
-                      overscanColumnCount={overscanColumnCount}
-                      overscanRowCount={overscanRowCount}
-                      rowCount={this.list.length}
-                      rowHeight={rowHeight}
-                      width={width}
-                    />
-                  </div>
+        <InfiniteLoader
+          isRowLoaded={this.isRowLoaded}
+          loadMoreRows={this.loadMoreRows}
+          rowCount={this.state.rowIdx}
+          threshold={1}
```

Comment:
1 seems pretty low, when scrolling the user will almost always have to wait. Why not use the default of 15?

Also, it would be nice to have this in a constant.